state/megawatcher_internal_test.go

```diff
@@ -503,6 +506,77 @@ func (s *storeManagerStateSuite) TestChanged(c *gc.C) {
 			c.Assert(err, jc.ErrorIsNil)
 			err = u.AssignToMachine(m)
 			c.Assert(err, jc.ErrorIsNil)
+			err = u.OpenPort("tcp", 4242)
+			c.Assert(err, jc.ErrorIsNil)
+
+			return testCase{
```

Comment:
FWIW, this test is becoming huge and unmanageable, we should split it to smaller (saner?) blocks.